modules/core/src/lib/layer-manager.js

```diff
@@ -216,8 +216,8 @@ export default class LayerManager {
     const reason = this.needsUpdate();
     if (reason) {
       this.setNeedsRedraw(`updating layers: ${reason}`);
-      // HACK - Call with a copy of lastRenderedLayers to trigger a full update
-      this.setLayers([...this.lastRenderedLayers]);
+      // Force a full update
+      this.setLayers(this.lastRenderedLayers, true);
```

Comment:
Nit:

The use of boolean parameters is generally considered evil as it tends to create very hard to find maintenance gotchas down the road and subverts self documenting aspects of an API. 

A simple way around to ameliorate this is to name the parameter in the calling code.

```
  const FORCE_UPDATE = true;
  this.setLayers(this.lastRenderedLayers, FORCE_UPDATE);
```

Although that will probably trigger reviewers to routinely request the constant be lifted to the top which would not really make sense, so maybe just:

```
  const forceUpdate = true;
  this.setLayers(this.lastRenderedLayers, forceUpdate);
```